android/lib/src/main/java/com/ern/api/impl/core/LaunchConfig.java

```diff
@@ -252,4 +266,26 @@ public void setReplace(boolean replace) {
     public boolean shouldReplace() {
         return mReplace;
     }
+
+    /**
+     * Use this when you don't need each fragment to register it's own navigation request handlers.
+     * When true: {@link com.ern.api.impl.navigation.ReactNavigationViewModel} will be created with activity scope instead of fragment scope.
+     * The activity will then delegate the navigation requests to the corresponding fragments via {@link com.ern.api.impl.navigation.NavigationRouteHandler} implementation.
+     * The {@link ElectrodeBaseActivityDelegate} will take the current instance of the {@link Fragment} that is loaded inside {@link LaunchConfig#getFragmentContainerId()}. Assumption is that this fragment implements {@link NavigationRouteHandler}
+     * Pass {@link RouteHandlerProvider} If you would like to provide your own implementation of providing the fragment instance, valid only if #value is true.
```

Comment:
No real action here, but a todo for an eventual cleanup: You probably noticed that these five lines are a one-to-one copy+paste from lines 97-101.

Reply:
removed.